### PR TITLE
Devops-1376-Brief introduction to oneTakeda organization with Readme

### PR DESCRIPTION
**Pull request by gayanak**
Devops-1376-Brief introduction to oneTakeda organization with Readme

This repo has fewer links compared to the .github-private because this is public facing and its assumed they might not have access them anyway. 

My thought process for the document is to have brief two lines on the following:

What is Onetakeda
What is the motivation behind the project
What problem we solve
Why us/this-project
How to start off
Open for suggestions and recommendations. Thanks

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -1,18 +1,12 @@
...
+OneTakeda is an organization within GitHub that provides enterprise standard tools & expertise to onboard your projects & accelerate your delivery.
+
+The main motivation behind this organization is to align different business units with the industry standards and avoid repetitive solutions for the same set of collective problems. 
+We make sure there is never a version mismatch on your deliverables by relying on GitHub version control features. This solution also supports distributed workforce (employees, consultants, partners, etc.) which is controlled by access policies with specified permissions to users and repositories.
 
-## &#x270d; Articles & Walkthroughs
+why would you want to reinvent the wheels, unless you plan on learning more about wheels and worse would be if you encounter a flat tire :smile: using our project will save engineers time and effort as we automate all your software workflows, Build, test, and deploy your code in a secure cloud environment.
```
Comment (jgleonard-takeda) [inline profile/README.md:8]: I like the sentiment here, but needs a little cleanup with capitalization, shorter sentences, and maybe a little more formal.

File: profile/README.md [hunk 2]
```diff
@@ -1,18 +1,12 @@
...
+**How do I access oneTakeda GitHub**?
 
-<!-- DEVOPS-PAGES-LIST:START -->
-- [DevOps @ Takeda - Home (sharepoint.com)](https://mytakeda.sharepoint.com/sites/DevOps)
-- [DevOps Team General Inquiry Form (sharepoint.com)](https://mytakeda.sharepoint.com/sites/DevOps/SitePages/General_Inquiry.aspx)
-- [DevOps Service Request Automation - Beta (sharepoint.com)](https://mytakeda.sharepoint.com/sites/DevOps/SitePages/DevOps-Service-Request-Automation-Beta.aspx)
-- [DevOps Capabilities Demo Repository](https://github.com/oneTakeda/devops-demo-app)
-<!-- DEVOPS-PAGES-LIST:END -->
\ No newline at end of file
+Users must first create their oneTakeda GitHub account through Okta SSO. Please visit this page to learn how to create your GitHub account [Github New User Access Guide](https://mytakeda.sharepoint.com/:p:/r/sites/DevOps/_layouts/15/Doc.aspx?sourcedoc=%7B20F91A70-6FB5-48D8-BB79-9009C9F16CC8%7D&file=GitHub%20New%20User%20Access%20Guide.pptx&action=edit&mobileredirect=true).
```
[the remark this answers is not in the document]
Comment (jgleonard-takeda) [reply profile/README.md:12]: @gayanak not sure, @mikefede what do you think?

[the remark this answers is not in the document]
Comment (mikefede) [reply profile/README.md:12]: Sorry for the delayed response. We don't have any publicly available docs and I agree with Jon that's its probably not a good idea to include an internal SharePoint link on a public facing page although I don't think there is any sensitive information that non-Takeda users could get from the URL. 

Here are the important points I think we should include:

- They should use a GitHub account they create for use at Takeda (not an existing account they use outside of Takeda, many people ignore this and we don't have a good way of enforcing that they create a separate account)
- The account should include their first and last name so it's easy for us to identify them (again many people ignore this and we can't enforce it)
- They should log in via Okta, which will link their GitHub account to their Takeda SSO identity.

File: profile/README.md [hunk 3]
```diff
@@ -1,18 +1,12 @@
...
 
-How do I access GitHub?
-Users must first create their oneTakeda GitHub account through Okta SSO. Please visit the [DevOps Visual Guides](https://mytakeda.sharepoint.com/sites/DevOps/SitePages/DevOps_Visual_Guides.aspx) page to learn how to create your GitHub account.
+OneTakeda is an organization within GitHub that provides enterprise standard tools & expertise to onboard your projects & accelerate your delivery.
+
+In this organization, we aim to align different business units with industry standards and avoid repetitive solutions for a set of collective problems. We utilize the Github version control features to ensure there is never a version mismatch on deliverables. This solution also supports distributed workforce (employees, consultants, partners, etc.) which is controlled by access policies with specified permissions to users and repositories.
 
-## &#x270d; Articles & Walkthroughs
+Why reinvent the wheels unless you learn more about wheels and worse still, a flat tire :smile:
```
Comment (jgleonard-takeda) [inline profile/README.md:7]: This still doesn't read quite right.  Maybe we take this line out.

**Review by jgleonard-takeda — APPROVED** [review]

**Pull request merged**